domain/secret/state/state.go

```diff
@@ -608,19 +628,21 @@ WHERE sm.secret_id = $secretID.id
 	return nil
 }
 
-// markObsoleteRevisions obsoletes the revisions and sets the pending_delete to true in the secret_revision table
-// for the specified secret if the revision is not the latest revision and there are no consumers for the revision.
+// markObsoleteRevisions obsoletes the revisions and sets the pending_delete
+// to true in the secret_revision table for the specified secret if the
+// revision is not the latest revision and there are no consumers for the
+// revision.
 func (st State) markObsoleteRevisions(ctx context.Context, tx *sqlair.TX, uri *coresecrets.URI) error {
 	query, err := st.Prepare(`
 WITH in_use AS (
     -- revisions that have local consumers.
     SELECT DISTINCT current_revision FROM secret_unit_consumer suc
     WHERE  suc.secret_id = $secretRef.secret_id
-UNION
+    UNION
```

Comment:
I really prefer the UNION alignment as it was as it better visually separates the different sql selects